tsconfig.json

```diff
@@ -1,14 +1,18 @@
 {
   "compilerOptions": {
-    "declaration": false,
+    "allowJs": true,
+    "allowSyntheticDefaultImports": true,
+    "composite": true,
+    "declaration": true,
     "emitDecoratorMetadata": true,
     "experimentalDecorators": true,
+    "jsx": "react",
     "lib": ["es6", "dom"],
     "module": "es6",
     "moduleResolution": "node",
     "sourceMap": true,
-    "target": "es5",
-    "jsx": "react"
+    "strict": false,
```

Comment:
`"strict": true` enables the good stuff

Reply:
instead of 

```
"strict": true
```

the plan is to explicitly state options as listed below

```
"noImplicitAny": false,
"noImplicitThis": true,
"alwaysStrict": true,
"strictBindCallApply": true,
"strictNullChecks": true,
"strictFunctionTypes": true,
"strictPropertyInitialization": true,
```

thank you @alexkrolick for helping me reason through this yesterday! 🙏🏽